internals/core_models.py

```diff
@@ -29,6 +29,17 @@
 class FeatureEntry(ndb.Model):  # Copy from Feature
   """This is the main representation of a feature that we are tracking."""
 
+  # Fields that should not be mutated by user edit requests.
+  FIELDS_IMMUTABLE_BY_USER = frozenset([
+    'id',
+    'created',
+    'creator_email',
+    'outstanding_notifications',
+    'deleted',
+    'star_count',
+    'feature_type',
+  ])
```

Comment:
It might be worth double-checking that these are the only fields we would like to protect from users with traditional edit access (non-admins).

Reply:
accurate_as_of?  We might want the API-centric version of the accurate_as_of confirmation page to be able to set it, but it should be the server that determines the value rather than something passed in.

updated and updater_email.  These will be overwritten by your code, but it would still be better to block them in case of future code changes and because blocking them will give an error message to clients that mistakenly try to set these fields.